src/renderer/components/ft-video-player/ft-video-player.js

```diff
@@ -364,6 +364,12 @@ export default defineComponent({
           await this.determineDefaultQualityLegacy()
         }
 
+        if (this.format === 'audio') {
+          const controlBarItems = this.dataSetup.controlBar.children
```

Comment:
Please help this goldfish memory Pikachu
I don't even remember `Array#splice`!
```suggestion
          // hide PIP button for audio format
          const controlBarItems = this.dataSetup.controlBar.children
```

Reply:
I have tested locally and it works though